### ISSUE TITLE
Consider adding tags to historical posts

### ISSUE THREAD

**Issue by jduckles** [datacarpentry/datacarpentry.github.io#494]
Based on a Slack thread:
![image](https://user-images.githubusercontent.com/119403/62583253-89d52b00-b903-11e9-8711-66551f6561db.png)

It appears all posts have no tags/categories at the moment, so they would have to be added.

**Comment by angela-li** [comment]
Thanks @jduckles for opening this issue based on my question. It would be great to be able to search Data Carpentry posts by tag. Right now, it's very difficult to find relevant/related posts without scrolling through all the content chronologically, which makes finding useful information difficult for instructors, organizers, and hosts. This could also be useful for search engine optimization!

**Comment by fmichonneau** [comment]
@serahrono I'm tagging you just in case you hadn't seen this. I know you've been thinking about tags for the other blogs.

**Comment by serahrono** [comment]
This is a great idea and we will work towards making tags available on https://librarycarpentry.org, https://datacarpentry.org and https://softwarecarpentry.org soon. Thanks @angela-li for the suggestion, @jduckles for opening the issue and @fmichonneau for flagging it. 

Please leave this issue open and we'll reference it as we make updates.

**Comment by maneesha** [comment]
@fmichonneau  is this addressed by the work you and @serahrono  are doing? Can we close this here?

**Comment by fmichonneau** [comment]
no, let's leave this open. For now, I just opened a PR that allows us to visualize blog posts by tags but none of the DC blog posts have tags. We'll be able to close this issue when the posts will be tagged I think on October 4th?

**Comment by serahrono** [comment]
hi @angela-li and @jduckles, work around this is still in progress, and we've gotten contributions from community and staff this month. There are still posts that need our attention, let me know if you have some time to dive into #499 this week, happy to co-work if you like

**Comment by maneesha** [comment]
@kmomar @acrall We have a number of issues about tags on blog post that have been open for up to two years. Do you want to keep these open?

**Comment by kmomar** [comment]
@maneesha @acrall and I will address this and other blog tag issues in an upcoming Reading and Reflection day

**Comment by acrall** [comment]
@kmomar This has been completed and can be closed, correct?
